docs/en-US/component/rate.md

```diff
@@ -84,6 +84,18 @@ Read-only Rate is for displaying rating score. Half star is supported.
 
 rate/readonly
 
+## CustomStyles
+
+Now you can set custom style for rate component.
+Use `css/scss` language to change the global or local color. We set some global color variables: `--el-rate-void-color`, `--el-rate-star-color`, `--el-rate-disable-void-color`, `--el-rate-text-color`. You can use like: `:root { --el-rate-void-color: red; --el-rate-star-color: blue; }`.
+
+### Default Variables
+
+- --el-rate-void-color:  \#c6d1de
```

Comment:
Hey, a space should to be deleted. Only lint pass, i can merge.

Reply:
OK